partners_api/cian_api.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "geometry/latlon.hpp"
+#include "geometry/rect2d.hpp"
+
+#include "base/worker_thread.hpp"
+
+#include <cstdint>
+#include <functional>
+#include <string>
+#include <vector>
+
+namespace cian
+{
+std::string const kBaseUrl = "https://api.cian.ru/rent-nearby/v1";
```

Comment:
So what? Imagine two individual cpp files, each includes cian_api.hpp. So, you have two definitions of the same constant, with the same name.